spec.html

```diff
@@ -1,19 +1,86 @@
+      <li>is an ordinary object.</li>
+      <li>has a [[Prototype]] internal slot whose value is the intrinsic object <emu-xref href="#sec-properties-of-the-object-prototype-object" id="_ref_5232"><a href="#sec-properties-of-the-object-prototype-object">%ObjectPrototype%</a></emu-xref>.</li>
+      <li>is not a <emu-xref href="#function-object" id="_ref_5233"><a href="#function-object">function object</a></emu-xref>.</li>
+      <li>does not have a [[Construct]] internal method; it cannot be used as a <a href="#constructor">constructor</a> with the <code>new</code> operator.</li>
+      <li>does not have a [[Call]] internal method; it cannot be invoked as a function.</li>
+    </ul>
+    <emu-note><span class="note">Note</span><div class="note-contents">
+      <p>In this specification, the phrase “the Number value for <var>x</var>” has a technical meaning defined in  <emu-xref href="#sec-ecmascript-language-types-number-type" id="_ref_502"><a href="#sec-ecmascript-language-types-number-type">6.1.6</a></emu-xref>.</p>
+    </div></emu-note>
+
+    <emu-clause id="sec-function-properties-of-the-math-object">
+      <h1><span class="secnum">20.2.2</span>Function Properties of the Math Object</h1>
+      <p>Each of the following <code>Math</code> object functions applies the <emu-xref aoid="ToNumber" id="_ref_5235"><a href="#sec-tonumber">ToNumber</a></emu-xref> abstract operation to each of its arguments (in left-to-right order if there is more than one). If <emu-xref aoid="ToNumber" id="_ref_5236"><a href="#sec-tonumber">ToNumber</a></emu-xref> returns an <emu-xref href="#sec-completion-record-specification-type" id="_ref_5237"><a href="#sec-completion-record-specification-type">abrupt completion</a></emu-xref>, that <emu-xref href="#sec-completion-record-specification-type" id="_ref_5238"><a href="#sec-completion-record-specification-type">Completion Record</a></emu-xref>
+ is immediately returned. Otherwise, the function performs a computation
+ on the resulting Number value(s). The value returned by each function
+is a Number.</p>
+      <ins class="block">
+        <emu-clause id="sec-math.get-random-values">
+          <h1><span class="secnum">20.2.2.36</span>Math.getRandomValues ( <var>typedArray</var> )</h1>
+          <p>Fills the array <var>typedArray</var> with cryptographically strong random values.</p>
+          <emu-alg>
+            <ol>
+              <li><emu-xref aoid="ValidateSharedIntegerTypedArray" id="_ref_7642"><a href="#sec-validatesharedintegertypedarray">ValidateSharedIntegerTypedArray</a></emu-xref>(<var>typedArray</var>).</li>
+              <li>Let <var>srcByteLength</var> be <var>typedArray</var>.[[ByteLength]].</li>
+              <li>If <var>srcByteLength</var> > 65536, then throw a <emu-val>RangeError</emu-val> exception.</li>
+              <li>Overwrite all elements of <var>typedArray</var> with cryptographically random values of the appropriate type.</li>
```

Comment:
"cryptographically _secure_ random value", but it's probably too early for these kinds of nits.